controllers/grid/settings/series/form/SeriesForm.php

```diff
@@ -78,7 +80,8 @@ public function initData()
         $categoryIds = [];
         if ($seriesId) {
             $series = $seriesDao->getById($seriesId, $press->getId());
-            $categoryIds = array_map('getId', $seriesDao->getCategories($seriesId, $press->getId()));
+            error_log(print_r($seriesDao->getCategories($seriesId, $press->getId()),true));
```

Comment:
Looks like a forgotten debug.

Reply:
These are draft PRs and not ready for a review yet!

Reply:
I saw it, I was just curious/passing by :)

Reply:
OK! I'm really just preparing these via Github for the sake of a complete Travis run; they'll get a read-over before they're merged.